output/cloud/output.go

```diff
@@ -17,11 +19,12 @@ import (
 	"go.k6.io/k6/output"
 	cloudv2 "go.k6.io/k6/output/cloud/expv2"
 	"go.k6.io/k6/usage"
-	"gopkg.in/guregu/null.v3"
 )
 
-// TestName is the default k6 Cloud test name
-const TestName = "k6 test"
```

Comment:
I believe it used in [k6-operator](https://github.com/grafana/k6-operator/blob/b5505372ca66fb028b6f1ac3c81d03316cfe504d/pkg/cloud/types.go#L41) so we need to coordinate with @yorugac

Reply:
Ah, no I am wrong here, please excuse the noise